packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.vue

```diff
@@ -74,6 +74,7 @@ watch(
       :class="{ 'onyx-switch__input': true, 'onyx-switch__loading': props.loading }"
       type="checkbox"
       :aria-label="props.hideLabel ? props.label : undefined"
+      :title="props.hideLabel ? props.label : undefined"
```

Comment:
did you try it out? somehow (only) the switch does not show the label when I hover on it in the 
http://localhost:6006/?path=/story/components-onyxswitch--default&args=hideLabel:!true

Reply:
also, we don't have a storybook pre-defined example here like on the other components for the "hidden label" example. not sure if we always need it but it might be better for consistency ¯\_(ツ)_/¯

Reply:
fixed